certified-connectors/GroupMgr/apiDefinition.swagger.json

```diff
@@ -1,982 +1,987 @@
 {
-   "swagger":"2.0",
-   "info":{
-      "title":"GroupMgr",
-      "description":"A powerful set of tools, for both IT managers and users, to efficiently create, manage, browse and analyse all groups in your Microsoft 365 environment.",
-      "version":"2.4",
-      "contact":{
-         "name":"GroupMgr Helpdesk",
-         "url":"https://www.vanroey.be/en/support/",
-         "email":"support@groupmgr.com"
-      }
-   },
-   "host":"vragroupmanagerapprovalflow-h8tvt-uzi02fbsspubp3eg.azurewebsites.net",
-   "basePath":"/",
-   "schemes":[
-      "https"
-   ],
-   "x-ms-connector-metadata":[
-      {
-         "propertyName":"Website",
-         "propertyValue":"https://groupmgr.com/"
-      },
-      {
-         "propertyName":"Privacy policy",
-         "propertyValue":"https://groupmgr.com/privacy"
-      },
-      {
-         "propertyName":"Categories",
-         "propertyValue":"IT Operations;Productivity"
-      }
-   ],
-   "consumes":[
-      
-   ],
-   "produces":[
-      
-   ],
-   "paths":{
-      "/api/webhookrequest/GroupRequested":{
-         "x-ms-notification-content":{
-            "description":"Caught group request.",
-            "schema":{
-               "$ref":"#/definitions/GroupCompact"
+    "swagger": "2.0",
+    "info": {
+        "title": "GroupMgr",
+        "description": "A powerful set of tools, for both IT managers and users, to efficiently create, manage, browse and analyse all groups in your Microsoft 365 environment.",
+        "version": "2.5",
+        "contact": {
+            "name": "GroupMgr Helpdesk",
+            "url": "https://www.vanroey.be/en/support/",
+            "email": "support@groupmgr.com"
+        }
+    },
+    "host": "vragroupmanagerapprovalflow-h8tvt-uzi02fbsspubp3eg.azurewebsites.net",
+    "basePath": "/",
+    "schemes": [
+        "https"
+    ],
+    "x-ms-connector-metadata": [{
+            "propertyName": "Website",
+            "propertyValue": "https://groupmgr.com/"
+        }, {
```

Comment:
@Jwaegebaert Please fix indentation issues in the swagger as properties are not aligned properly.

Reply:
@sriyen-msft  Could you elaborate a bit more about this issue. The selected part above looks like the indentations are properly aligned.

Reply:
@Jwaegebaert Each and of property of each section in the swagger should be properly aligned with tab index. If you see this section, the brackets are not properly aligned as the opening and closing braces aligned on the same line. You can refer [a connector](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/SAS%20Decisioning/apiDefinition.swagger.json#L13) in the open source repo for reference.

Reply:
@sriyen-msft Thanks for the elaboration. The changes are commited.